spec/date.html

```diff
@@ -13,20 +13,22 @@ <h1>The Temporal.Date Constructor</h1>
       <li>is the intrinsic object <dfn>%Temporal.Date%</dfn>.</li>
       <li>creates and initializes a new Temporal.Date object when called as a constructor.</li>
       <li>is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified Temporal.Date behaviour must include a super call to the %Temporal.Date% constructor to create and initialize subclass instances with the necessary internal slots.</li>
+      <li>has a `"length"` property whose value is 4.</li>
     </ul>
 
     <emu-clause id="sec-temporal.date">
-      <h1>Temporal.Date ( _isoYear_, _isoMonth_, _isoDay_ )</h1>
+      <h1>Temporal.Date ( _isoYear_, _isoMonth_, _isoDay_, [ _calendar_ ] )</h1>
       <p>
         When the `Temporal.Date` function is called, the following steps are taken:
       </p>
       <emu-alg>
         1. If NewTarget is *undefined*, then
           1. Throw a *TypeError* exception.
+        1. If _calendar_ is *undefined*, set _calendar_ to ? GetDefaultCalendar().
```

Comment:
We should call this GetISO8601Calendar or GetConstructorDefaultCalendar now, probably in the polyfill code as well.

Reply:
Let me know which name is more preferable to you, I don't have any opinions here. I'll make that change in both place.s

Reply:
I wouldn't consider it a blocker for this PR, by the way, so if you want to defer it to a follow up that's fine with me. I think GetISO8601Calendar is most straightforward.

Reply:
#984